.github/workflows/openactive-test-suite.yml

```diff
@@ -63,10 +63,10 @@ jobs:
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: 3.1.419
-    - name: Setup Node.js 14.x
+    - name: Setup Node.js 18.17.1
```

Comment:
Why 18.17.1 rather than 18.19.1? Is it what other OA apps are using?

Reply:
Yes exactly